test/pass.jsx

```diff
@@ -70,6 +70,7 @@ class MyComponent extends Component {
 const Fragments = (props) => (
   <Fragment>
     <Banner>{props.message}</Banner>
+    <span>Some default text</span>
```

Comment:
This allows it to be valid according to the new `react/jsx-no-useless-fragment`.

This is autofixable to remove useless fragments, but in this case I think we want to keep the fragment in the test!